MIGRATION.md

```diff
@@ -209,7 +230,7 @@ Many Storybook packages are now available as ESModules in addition to CommonJS.
 
 ```js
 // In your jest config
-transformIgnorePatterns: ['/node_modules/(?!@storybook)']
+transformIgnorePatterns: ['/node_modules/(?!@storybook)'];
```

Comment:
_Tiny_ nit, but this line of config is likely either in an object or JSON, so the semi-colon is a syntax error.

More importantly, should we update the text above this to inform people that this isn't necessary in 6.3?

Reply:
Good catch, updated!